### PR TITLE
games

### PR DESCRIPTION
**Pull request by Khalidir**

### CODE REVIEW

File: app/views/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+      <h3><%= title %></h3>
+        <%= simple_form_for(game) do |f| %>
+        <%= f.input :title %>
+        <%= f.input :genre %>
+        <%= f.submit "Select", class: "btn btn-outline-dark" %> <%= link_to "Back to games", games_path, class: "btn btn-outline-dark" %>
+      <% end %>
+    </div>
+   </div>
+  </div>
+</div>
```
Comment (vladiangelov) [inline app/views/index.html.erb]: That is a simple form, can you create an index view instead?

File: app/views/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+      <h3><%= title %></h3>
+        <%= simple_form_for(game) do |f| %>
+        <%= f.input :title %>
+        <%= f.input :genre %>
+        <%= f.submit "Select", class: "btn btn-outline-dark" %> <%= link_to "Back to games", games_path, class: "btn btn-outline-dark" %>
+      <% end %>
+    </div>
+   </div>
+  </div>
+</div>
```
Comment (vladiangelov) [inline app/views/show.html.erb]: That is also a simple form, can you create a show view instead?

File: app/controllers/games_controller.rb [hunk 3]
```diff
@@ -1,2 +1,23 @@
 class GamesController < ApplicationController
+    before_action :set_game
```
Comment (vladiangelov) [inline app/controllers/games_controller.rb]: Make sure you have the right indentation here.

Comment (brycole) [reply app/controllers/games_controller.rb]: before_action :set_Game should also be limited to only the methods needed. e.g. "show", "update", "destroy"

**Pull request merged**